curriculum/challenges/english/25-front-end-development/quiz-form-validation-with-javascript/66edd3403d7077eece6dc4b6.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
 #### --text--
 
-Placeholder question
+Which method is used to check if an input element's value satisfies the validation rules?
 
 #### --distractors--
 
-Placeholder distractor 1
+`validateInput()`
 
 ---
 
-Placeholder distractor 2
+`isValid()`
 
 ---
 
-Placeholder distractor 3
+`checkValue()`
 
 #### --answer--
 
-Placeholder answer
+`checkValidity()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens if the `checkValidity()` method finds an invalid input?
 
 #### --distractors--
 
-Placeholder distractor 1
+The form is automatically submitted
 
 ---
 
-Placeholder distractor 2
+Nothing happens
 
 ---
 
-Placeholder distractor 3
+The input value is cleared
 
 #### --answer--
 
-Placeholder answer
+The browser displays a validation message
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which property is used to customize the message shown during validation failure?
 
 #### --distractors--
 
-Placeholder distractor 1
+`setErrorMessage`
 
 ---
 
-Placeholder distractor 2
+`setValidationMessage`
 
 ---
 
-Placeholder distractor 3
+`customMessage`
 
 #### --answer--
 
-Placeholder answer
+`setCustomValidity()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the default validation behavior for required fields when the form is submitted?
 
 #### --distractors--
 
-Placeholder distractor 1
+The form is submitted even if required fields are empty
 
 ---
 
-Placeholder distractor 2
+The form submits but gives a warning
 
 ---
 
-Placeholder distractor 3
+The form resets the input values
 
 #### --answer--
 
-Placeholder answer
+The browser stops the form submission if required fields are empty
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which actions does not cause an HTML form to be checked for errors?
 
 #### --distractors--
 
-Placeholder distractor 1
+Changing the input value
 
 ---
 
-Placeholder distractor 2
+Submitting the form
 
 ---
 
-Placeholder distractor 3
+Calling `reportValidity()`
 
 #### --answer--
 
-Placeholder answer
+Clicking a reset button
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does `checkValidity()` return when all form inputs are valid?
 
 #### --distractors--
 
-Placeholder distractor 1
+false
 
 ---
 
-Placeholder distractor 2
+undefined
 
 ---
 
-Placeholder distractor 3
+null
 
 #### --answer--
 
-Placeholder answer
+true
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+When does the `checkValidity()` method stop form submission?
```

Comment:
I don't believe `checkValidity()` does this. It does fire an invalid event on the input elements, but NOT on the form itself (and the input-level event has no default behaviour).